deploy/examples/keycloak/keycloak-with-experimental-settings.yaml

```diff
@@ -29,3 +29,6 @@ spec:
           - configMap:
               name: test-config
               mountPath: /test-config
+          - secret:
+              name: keytabs
```

Comment:
We probably need a small definition of an example Secret in this file (see how the ConfigMap is done).

Reply:
Ok, I added a example secret